source/simulator/core.cc

```diff
@@ -1781,20 +1837,13 @@ namespace aspect
           assemble_advection_system (AdvectionField::temperature());
           solve_advection(AdvectionField::temperature());
 
-          if (parameters.use_discontinuous_temperature_discretization
-              && parameters.use_limiter_for_discontinuous_temperature_solution)
-            apply_limiter_to_dg_solutions(AdvectionField::temperature());
-
```

Comment:
This change is independent of the other one. Can you separate this into a different pull request?